src/components/KeyboardShortcuts/index.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+
+import { ConnectedReduxProps } from '../../configureStore';
+import {
+  FileTree,
+  RelativePathPosition,
+  goToRelativeFile,
+} from '../../reducers/fileTree';
+import { actions as versionsActions } from '../../reducers/versions';
+import styles from './styles.module.scss';
+import { gettext } from '../../utils';
+
+const keys = ['k', 'j', 'e', 'c'];
+
+export type PublicProps = {
+  currentPath: string;
+  pathList: FileTree['pathList'];
+  versionId: number;
+};
+
+export type DefaultProps = {
+  _goToRelativeFile: typeof goToRelativeFile;
+};
+
+type Props = PublicProps & DefaultProps & ConnectedReduxProps;
+
+export class KeyboardShortcutsBase extends React.Component<Props> {
+  static defaultProps: DefaultProps = {
+    _goToRelativeFile: goToRelativeFile,
+  };
+
+  keydownListener = (event: KeyboardEvent) => {
+    const {
+      _goToRelativeFile,
+      currentPath,
+      dispatch,
+      pathList,
+      versionId,
+    } = this.props;
+
+    if (
+      !event.altKey &&
+      !event.ctrlKey &&
+      !event.metaKey &&
+      !event.shiftKey &&
+      keys.includes(event.key)
+    ) {
+      switch (event.key) {
+        case 'k':
+          dispatch(
+            _goToRelativeFile({
+              currentPath,
+              pathList,
+              position: RelativePathPosition.previous,
+              versionId,
+            }),
+          );
+          break;
+        case 'j':
+          dispatch(
+            _goToRelativeFile({
+              currentPath,
+              pathList,
+              position: RelativePathPosition.next,
+              versionId,
+            }),
+          );
+          break;
+        case 'e':
+          dispatch(
+            versionsActions.expandTree({
+              versionId,
+            }),
+          );
+          break;
+        case 'c':
+          dispatch(
+            versionsActions.collapseTree({
+              versionId,
+            }),
+          );
+          break;
+        default:
+      }
+    }
+  };
+
+  componentDidMount() {
+    document.addEventListener('keydown', this.keydownListener);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this.keydownListener);
+  }
+
+  render() {
+    return (
+      <div className={styles.KeyboardShortcuts}>
+        <h4>{gettext('Keyboard Shortcuts')}</h4>
+
+        <dl>
+          <dt>k</dt>
```

Comment:
It would be great to use [`kbd` tags](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd) for each key.

Reply:
Great idea!